inlang/source-code/sdk/src/storage/human-id/human-readable-id.ts

```diff
@@ -0,0 +1,27 @@
+import murmurhash3 from "murmurhash3js" // we use murmur for best distribution https://medium.com/miro-engineering/choosing-a-hash-function-to-solve-a-data-sharding-problem-c656259e2b54
```

Comment:
increase readability by moving the comment above the import, not next to the import

```suggestion
// we use murmur for best distribution https://medium.com/miro-engineering/choosing-a-hash-function-to-solve-a-data-sharding-problem-c656259e2b54
import murmurhash3 from "murmurhash3js"
```

Reply:
👍